projects/example.yaml

```diff
@@ -34,6 +34,8 @@ paths:
             /page:
               x-modules:
                 - path: v1/content.yaml
+                  options:
+                    response_cache_control: '{{options.purged_cache_control}}'
```

Comment:
`Response` is fairly generic. Some of our responses aren't purged, and have considerably shorter TTLs.

What do you think about keeping the name `purged_cache_control`?